getting_started/dialogs.rst

```diff
@@ -0,0 +1,144 @@
+Writing Dialogs
+================
+
+Dialogs are an essential part of narrative games. Whether it's about world
+building, fleshing out characters or presenting puzzles, dialogs are usually
+found in every narrative game.
+
+Escoria supports a feature-rich dialog system integrated in the
+:doc:`ESC language </scripting/z_esc_reference>`.
+
+Dialog concept
+~~~~~~~~~~~~~~
+
+Dialogs in Escoria are based on two parts:
+
+- The :doc:`say command </api/SayCommand>` to display text, show speech
+  animations and play voice files for a character
+- A dialog chooser that displays a list of options for the player to choose
+  while engaged in a dialog.
+
+A simple dialog
+~~~~~~~~~~~~~~~
+
+The following code shows a simple dialog:
+
+.. code-block::
+
+    #1
+    :talk worker
```

Comment:
`:talk` event does not need a parameter, since this script is already in the worker's ESC script.
Events parameters are useful for objects used ON or WITH.
For example, you could define multiple `:give` events, one per object you can give to the target:
```
:give apple
# do stuff

:give banana
# do other stuff
``` 

But for talk verb this is not useful.

Reply:
Ah right.

Reply:
I added more details to the paragraph as well.